splunk_connector.py

```diff
@@ -794,28 +819,49 @@ def _update_event(self, param):
         # 3. Update the provided Events ID
         request_body = {"ruleUIDs": ids}
 
-        if owner:
-            request_body['newOwner'] = owner
         if integer_status is not None:
             if int(integer_status) not in list(self._splunk_status_dict.values()):
-                return action_result.set_status(phantom.APP_ERROR, "Please provide a valid value in 'integer_status' action\
-                     parameter. Valid values: {}".format((', '.join(map(str, list(self._splunk_status_dict.values())))) ))
-            request_body['status'] = str(integer_status)
+                return action_result.set_status(
+                    phantom.APP_ERROR,
+                    "Please provide a valid value in 'integer_status' action\
+                     parameter. Valid values: {}".format(
+                        (", ".join(map(str, list(self._splunk_status_dict.values()))))
+                    ),
+                )
+            request_body["status"] = str(integer_status)
         elif status:
             if status not in self._splunk_status_dict:
                 if not status.isdigit():
                     return action_result.set_status(phantom.APP_ERROR, consts.SPLUNK_ERR_BAD_STATUS)
-                request_body['status'] = status
+                request_body["status"] = status
+            else:
+                request_body["status"] = self._splunk_status_dict[status]
+
+        if integer_disposition is not None:
+            if int(integer_disposition) not in list(self._splunk_disposition_dict.values()):
```

Comment:
```suggestion
            if int(integer_disposition) not in self._splunk_disposition_dict.values():
```

And isn't integer disposition an `int`at this point already, why cast?

Reply:
done